colors/1989.vim

```diff
@@ -52,7 +52,7 @@ call <SID>set_hi("StatusLine", s:default_white, s:mid_gray, "bold")
 call <SID>set_hi("StatusLineNC", s:default_white, s:mid_gray, "NONE")
 call <SID>set_hi("Pmenu", s:none, s:none, "NONE")
 call <SID>set_hi("PmenuSel", s:none, s:dark_gray, "NONE")
-call <SID>set_hi("IncSearch", s:default_white, s:light_yellow, "NONE")
+call <SID>set_hi("IncSearch", s:default_gray, s:light_yellow, "NONE")
```

Comment:
Hi, thanks for the PR! I don't think `s:default_gray` is defined in this color scheme, do you mean `s:dark_gray`?